### PR TITLE
Expose enrollmentCompletionDeadline field in TotpSecret.

### PR DESCRIPTION
**Pull request by prameshj**
Also changed the demo app to display this field as a countdown.

cc @bhparijat @sam-gc

### CODE REVIEW

File: common/api-review/auth.api.md [hunk 1]
```diff
@@ -767,6 +767,7 @@ export interface TotpMultiFactorInfo extends MultiFactorInfo {
 export class TotpSecret {
     readonly codeIntervalSeconds: number;
     readonly codeLength: number;
+    readonly enrollmentCompletionDeadline: string;
```
Comment (bhparijat) [inline common/api-review/auth.api.md:770]: Don't understand why it was changed to this instead of finalizeEnrollmentBy

**Review by bhparijat — COMMENTED** [review]
Just 1 question

File: common/api-review/auth.api.md (hunk 1 above)
Comment (prameshj) [reply common/api-review/auth.api.md:770]: This is still pending API review, but IMO enrollmentCompletionDeadline sounds slightly clearer.

**Comment by prameshj** [comment]
Merging based on offline approval from @sam-gc. This is being merged to the feature branch, not master.

**Pull request merged**